incubator/kubernetes-vault/values.yaml

```diff
@@ -5,21 +5,35 @@
 replicaCount: 3
 image:
   repository: boostport/kubernetes-vault
-  tag: v0.4.8
+  tag: v0.5.2
   pullPolicy: Always
 app: kubernetes-vault
 service:
   dummyPort: 80
-vault:
-  address: http://vault:8200
-  token: change-this-value
-  vaultCertBackend: intermediate-ca
-  vaultCertRole: kubernetes-vault
+
+# Maps directly to https://github.com/Boostport/kubernetes-vault#configuration
+configMap:
+  vault:
+    addr: http://vault:8200
```

Comment:
According to the e2e tests, this setting tries to make it connect to a non-existing Service. I'm not entire sure what the proper solution here would be, but I think a template (like used a few lines below this, for `configMap.kubernetes.service`, might be a good solution.

Reply:
`kubernetes-vault` depends on running and properly configured instance of Vault.
I don't think it is possible to configure it inside the chart.

Reply:
I'm not sure what the easy solution for this would be. Is there another chart you can depend upon? Not sure if that would work as I expect it to, though.

Reply:
It might be possible to set up `incubator/vault` using hooks, but would require a lot of effort just to make it pass the tests.

Reply:
Vault requires a fair bit of setup as a prerequisite to this chart, so building a vault that can be automatically deployed and setup for this would probably be a big lift. We'd have top hope that dev mode would permit automatic initialization and unsealing for our purposes here.